help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce pardot;salesforce marketing cloud account engagement;salesforce pardot destination
```

Comment:
minor, but from now on let's please drop the `keywords` metadata. historically, this was used to improve search results on our documentation website, but this is not needed anymore following some website improvements.

Reply:
Assume across all streams i.e. destination, Source and Extension. Please let me know if otherwise.